cli/CHANGELOG.md

```diff
@@ -0,0 +1,51 @@
+# Changelog
+
+All notable changes to the Sample Plug-in for Zowe CLI will be documented in this file.
```

Comment:
This came from the sample, should be removed.

Reply:
Any changelogs you do want to leave, should be in the CHANGELOG.md in the root directory.